nautobot/core/tests/test_views.py

```diff
@@ -71,6 +73,26 @@ def test_search(self):
         response = self.client.get(f"{url}?{urllib.parse.urlencode(params)}")
         self.assertHttpStatus(response, 200)
 
+    def test_appropriate_models_included_in_global_search(self):
+        # Gather core app configs
+        existing_models = []
+        global_searchable_models = []
+        for app_name in ["circuits", "dcim", "extras", "ipam", "tenancy", "virtualization"]:
+            app_config = apps.get_app_config(app_name)
+            existing_models += [model._meta.model_name for model in app_config.get_models()]
+            global_searchable_models += app_config.searchable_models
+
+        # Remove those models that are not searchable
+        existing_models = [model for model in existing_models if model not in GLOBAL_SEARCH_EXCLUDE_LIST]
+        existing_models.sort()
+
+        # See if there are any models that are missing from global search
+        difference = [model for model in existing_models if model not in global_searchable_models]
+        if difference:
+            self.fail(
+                f'Existing model/models {",".join(difference)} are not included in the searchable_models attribute of the app config.\nIf you do not want the models to be searchable, please include them in the GLOBAL_SEARCH_EXCLUDE_LIST constant in nautobot.core.constants.'
```

Comment:
We really should wrap this to 120 characters. Maybe better to do in `develop` than here though?

Reply:
I opened a PR for this comment and the one below https://github.com/nautobot/nautobot/pull/5584, should we merge that before this is merged or after?

Reply:
That can live in develop for now and bubble up to next after 2.2.2, IMHO.